CMakeLists.txt

```diff
@@ -1,60 +1,38 @@
 ################################################################################
 #  THIS FILE IS 100% GENERATED BY ZPROJECT; DO NOT EDIT EXCEPT EXPERIMENTALLY  #
-#  Please refer to the README for information about making permanent changes.  #
+#  Read the zproject/README.md for information about making permanent changes. #
 ################################################################################
 
 ########################################################################
 # Project setup
 ########################################################################
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 2.8.12)
 project(filemq)
 enable_language(C)
 enable_testing()
 
 set(SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
+set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
+# Select flags
+if(MSVC)
+    SET(CMAKE_C_FLAGS_RELEASE "/O2")
+else()
+    SET(CMAKE_C_FLAGS_RELEASE "-O3")
+endif()
 
-########################################################################
-# determine version
-########################################################################
-foreach(which MAJOR MINOR PATCH)
-    file(
-        STRINGS
-        "${SOURCE_DIR}/include/filemq_library.h"
-        FILEMQ_VERSION_STRING REGEX
-        "#define FILEMQ_VERSION_${which}"
-    )
-    string(
-        REGEX MATCH
-        "#define FILEMQ_VERSION_${which} ([0-9_]+)"
-        FILEMQ_REGEX_MATCH
-        "${FILEMQ_VERSION_STRING}"
-    )
-    if (NOT FILEMQ_REGEX_MATCH)
-        message(
-            FATAL_ERROR
-            "failed to parse FILEMQ_VERSION_${which} from filemq.h"
-        )
-    endif()
-    set(FILEMQ_${which}_VERSION ${CMAKE_MATCH_1})
```

Comment:
I believe the new zproject's approach to this problem is much more elegant so this should be removed